# Copyright 2021, Pick-It NV.
# All rights reserved.

---
# Array with object detections.
ObjectArray objects
